### PR TITLE
support for the larger noisy student models

### PR DESCRIPTION
**Pull request by ThomasHenckel**
Added include_top option to load_efficientnet, making it possible to make notop models for transfer learning

Added efficientnet-b6, b7 and l2

### CODE REVIEW

**Comment by qubvel** [comment]
Hi, thanks a lot for PR, did you save weights for converted models?

**Comment by ThomasHenckel** [comment]
Yes, i converted the weight file for the noisy student models, i planned upload them here: https://github.com/ThomasHenckel/efficientnet_noisy_student_keras_transfer_learning
I just havent gotten transfer learning working on the l2 model yet

**Comment by MichaelKarpe** [comment]
Hi @qubvel, thanks for this repo! I used your converter script a few months ago to generate `autoaugment` and `advprop` weights (`top` and `notop`), I was thinking you may be interested in adding these weights to your efficientnet repository. Please find them in [here](https://www.kaggle.com/mika30/efficientnet-weights-for-keras).

> Yes, i converted the weight file for the noisy student models, i planned upload them here: https://github.com/ThomasHenckel/efficientnet_noisy_student_keras_transfer_learning
> I just havent gotten transfer learning working on the l2 model yet

Hi @ThomasHenckel, thanks for the support for larger noisy student models! Are you still considering uploading weights files in the GitHub repo you mentioned above? I look forward to being able to use them in @qubvel efficientnet repo!

**Comment by qubvel** [comment]
Hi @MichaelKapre
For now default 'imagenet' weights are actually imagenet weight trained with autoaugment. 
I have also already add noisy student trained weights.
I will consider adding 'advprop' weights, thanks a lot for sharing!

File: scripts/load_efficientnet.py [hunk 1]
```diff
@@ -32,6 +32,9 @@ def _get_model_by_name(name, *args, **kwargs):
         'efficientnet-b3': efficientnet.keras.EfficientNetB3,
         'efficientnet-b4': efficientnet.keras.EfficientNetB4,
         'efficientnet-b5': efficientnet.keras.EfficientNetB5,
+        'efficientnet-b6': efficientnet.keras.EfficientNetB6,
+        'efficientnet-b7': efficientnet.keras.EfficientNetB7,
```
Comment (henrique) [inline scripts/load_efficientnet.py:36]:
```suggestion
        'efficientnet-b7': efficientnet.keras.EfficientNetB7,
        'efficientnet-b8': efficientnet.keras.EfficientNetB8,
```
missing **EfficientNetB8**

**Review by omerferhatt — APPROVED** [review]

**Pull request closed without merging**